diff --git a/node_modules/react-native/Libraries/Utilities/deepFreezeAndThrowOnMutationInDev.js b/node_modules/react-native/Libraries/Utilities/deepFreezeAndThrowOnMutationInDev.js
--- a/node_modules/react-native/Libraries/Utilities/deepFreezeAndThrowOnMutationInDev.js
+++ b/node_modules/react-native/Libraries/Utilities/deepFreezeAndThrowOnMutationInDev.js
@@ -27,48 +27,11 @@
  * Freezing the object and adding the throw mechanism is expensive and will
  * only be used in DEV.
  */
 function deepFreezeAndThrowOnMutationInDev<T: {...} | Array<mixed>>(
   object: T,
 ): T {
-  if (__DEV__) {
-    if (
-      typeof object !== 'object' ||
-      object === null ||
-      Object.isFrozen(object) ||
-      Object.isSealed(object)
-    ) {
-      return object;
-    }
-
-    // $FlowFixMe[not-an-object] `object` can be an array, but Object.keys works with arrays too
-    const keys = Object.keys((object: {...} | Array<mixed>));
-    // $FlowFixMe[method-unbinding] added when improving typing for this parameters
-    const hasOwnProperty = Object.prototype.hasOwnProperty;
-
-    for (let i = 0; i < keys.length; i++) {
-      const key = keys[i];
-      if (hasOwnProperty.call(object, key)) {
-        Object.defineProperty(object, key, {
-          get: identity.bind(null, object[key]),
-        });
-        Object.defineProperty(object, key, {
-          set: throwOnImmutableMutation.bind(null, key),
-        });
-      }
-    }
-
-    Object.freeze(object);
-    Object.seal(object);
-
-    for (let i = 0; i < keys.length; i++) {
-      const key = keys[i];
-      if (hasOwnProperty.call(object, key)) {
-        deepFreezeAndThrowOnMutationInDev(object[key]);
-      }
-    }
-  }
   return object;
 }
 
 /* $FlowFixMe[missing-local-annot] The type annotation(s) required by Flow's
  * LTI update could not be added via codemod */
